its/plugin/src/test/java/com/sonar/python/it/plugin/TestReportTest.java

```diff
@@ -26,6 +26,8 @@
 import com.sonar.orchestrator.Orchestrator;
 import com.sonar.orchestrator.build.BuildResult;
 import com.sonar.orchestrator.build.SonarRunner;
+
+import org.junit.Before;
```

Comment:
@ivandalbosco which settings for import formatting do you have? It should be like this:
import all other imports
<blank line>
import static all other imports

(I'm asking because i see blank line on previous line)